File: app/src/main/java/com/kroraina/easyreader/modules/sheetlist/detail/BookListDetailContract.java
package com.kroraina.easyreader.modules.sheetlist.detail;

import com.kroraina.easyreader.base.mvp.BaseContract;

public interface BookListDetailContract {

    interface View extends BaseContract.BaseView{
        void finishRefresh(BookListDetailBean bean);
    }

    interface Presenter extends BaseContract.BasePresenter<View>{
        void refreshBookListDetail(String detailId);
    }
}

File: app/src/main/java/com/kroraina/easyreader/modules/sheetlist/detail/BookListDetailContract.kt
package com.kroraina.easyreader.modules.sheetlist.detail

import com.kroraina.easyreader.base.mvp.BaseContract

interface BookListDetailContract {

    interface View : BaseContract.BaseView {
        fun finishRefresh(bean: BookListDetailBean)
    }

    interface Presenter : BaseContract.BasePresenter<View> {
        fun refreshBookListDetail(detailId: String)
    }
}
